benchmarks/flat_map_vs_inject.rb

```diff
@@ -0,0 +1,24 @@
+require 'benchmark/ips'
+require 'rspec/core/flat_map'
+
+words = %w[ foo bar bazz big small medium large tiny less more good bad mediocre ]
+
+Benchmark.ips do |x|
+  x.report("flat_map") do
+    words.flat_map(&:codepoints)
+  end
+
+  x.report("inject") do
+    words.inject([]) { |a, w| a + w.codepoints }
```

Comment:
For fair benchmark, I think this should be `a.concat(w.codepoints)` which doesn't create useless array instance for each iteration.